exercises/concept/ticket-please/.meta/Exemplar.elm

```diff
@@ -0,0 +1,67 @@
+module TicketPlease exposing (..)
+
+
+type Ticket
```

Comment:
I think it would be better to have these already defined for the user, probably in a supporting file. The Go exercise has an example of this (https://github.com/exercism/elm/tree/main/exercises/concept/go/src)

Reply:
Yep fair enough. This was a comment Erik left me so I'm passing it on. I think the concern is that a user might edit the code, and could then get in a muddle, and having it in a separate file makes it very clear that the code shouldn't be edited.

Reply:
Hi Jie, sorry to be a pain, but I think we should do this. It's a suggestion from Erik, and we have already done it for other exercises, so I think it is best to be consistent. We could maybe bring it up at a weekly call sometime, and see if we can address your point about the user experience being worse. Then we could apply the change to all the exercises at once to keep the consistent experience.